### PR TITLE
[java/de] Initial translation added.

### PR DESCRIPTION
**Pull request by JaXt0r**
This is an initial translation of the original Java_en markdown into german language.

### CODE REVIEW

File: de-de/java-de.html.markdown [hunk 1]
```diff
@@ -0,0 +1,491 @@
+---
+language: java
+contributors:
+    - ["Michael Dähnert", "http://github.com/JaXt0r"]
+filename: LearnJava-de.java
```
Comment (vendethiel) [inline de-de/java-de.html.markdown]: please add a `-de` suffix to the filename, and restore the original contributors (add yourself as a translator instead)

**Comment by vendethiel** [comment]
@JaXt0r ?

**Comment by JaXt0r** [comment]
Thanks for the reminder. I handled your hints and pushed them. Please review.

**Comment by vendethiel** [comment]
Thank you very much!

**Pull request merged**